epochs/hippieeg/config/snakebids.yml

```diff
@@ -0,0 +1,124 @@
+bids_dir: '/home/mcesped/projects/ctb-akhanf/cfmm-bids/Khan/epi_iEEG/ieeg'
+bids_dir_tsv: '/home/mcesped/projects/ctb-akhanf/cfmm-bids/Khan/clinical_imaging/epi_iEEG/derivatives/seega_coordinates'
```

Comment:
If this file requires another bids directory its better to just use a command line input like `--path-tsv` or whatever the file is called and you can provide the direct path to the file with wildcards which can be populated with wildcard inputs from the corresponding images automatically.

Reply:
Actually I realized you can define the path to this file with the `--derivatives` tag this is your best option as opposed to defining a new variable. this is your best option for defining the path to this file in a more flexible way.

Reply:
Actually, as I am using the custom path, this variable is not used. I was wondering, these flags (like '--derivatives') are already available by default or do I need to add them through the parse_args?